media/src/scss/ltdc-base/_button.scss

```diff
@@ -37,3 +37,33 @@
         @extend %user-auth;
     }
 }
+
+.overflow-menu {
+    @extend .btn;
+    width: 2.25rem;
+    height: 2.25rem;
+    background-color: transparent;
+    color: $gray-ltds-900;
+    border-radius: 50%;
+    margin-left: auto;
+  -webkit-transition: background-color 0.3s ease-out;
```

Comment:
We're using `autoprefixer`, so I don't believe these are needed. Not a blocking issue though.